$NetBSD$

Do not install the bundled image and XML libraries, which are not built.

--- CMakeLists.txt.orig	2026-10-24 00:46:49
+++ CMakeLists.txt	2026-10-24 00:47:01
@@ -494,14 +494,8 @@
 endif()
 
 # Installation targets
-set(MI_DEPEND
-  IlmImf IlmThread Imath Iex IexMath Half pugixml
-)
-
-list(APPEND MI_DEPEND struct-jit)
+set(MI_DEPEND struct-jit)
 
-list(APPEND MI_DEPEND png jpeg)
-
 if (WIN32)
   list(APPEND MI_DEPEND zlib)
 endif()
